src/ContactBundle.ts

```diff
@@ -14,9 +14,7 @@ export default class ContactBundle implements proto.ContactBundleV1 {
   }
 
   toBytes(): Uint8Array {
-    return proto.ContactBundleV1.encode({
-      keyBundle: this.keyBundle,
-    }).finish()
+    return this.keyBundle.toBytes()
```

Comment:
Returning the PublicKeyBundle is what is desired here for the time being. Once clients have updated we can have clients publish versioned bundles.